### DISCUSSION TITLE
Testing Observability Pipeline

### DISCUSSION THREAD

**Discussion by francisan** [open-telemetry/opentelemetry-java-instrumentation#12119] [Q&A]
Hi,

How I can test below pipeline using some sample payload. Want to test/ensure connector routes the request to corresponding pipeline based on the field 'name'.

receivers:
  webhookevent:
  
        endpoint: 0.0.0.0:1111
        path: /get/info
        
        
       
processors:
  transform/tr:
  
         error_mode: ignore
         log_statements:
          - context: log
            statements:
             - set(body, ParseJSON(body)) where body != nil
             - set(resource.attributes["name"], body["name"])
            
         
connectors:
  routing :

    table:
     - statement: route() where attributes["name"] == "abc"
       pipelines:
         - logs/abc
     - statement: route() where attributes["name"] == "xyz"
       pipelines:
         - logs/xyz            
        
 

exporters:
  logging:
  
    loglevel: debug

         
service:
  pipelines:
  
         
    logs/receivers:
      receivers: [webhookevent]
      processors: [transform/tr]
      exporters: [routing,logging]
      
    logs/abc:
      receivers: [routing]
      processors: []
      exporters: [logging]
      
    logs/xyz:
      receivers: [routing]
      processors: []
      exporters: [logging]

**Comment by laurit** [comment]
You are better off asking this question from the collector folks.

**Reply by francisan** [reply]
Thanks @laurit 
Posted in https://github.com/open-telemetry/opentelemetry-collector/issues/10977
